### PR TITLE
Use __unix virtual package

### PR DESCRIPTION
**Pull request by MetRonnie**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

Use the `__unix` [virtual package](https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-virtual.html) to try and enable [noarch](https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python)

### CODE REVIEW

**Comment by ngam** [comment]
@conda-forge/cylc-flow are you still unsure about noarch?

**Comment by ngam** [comment]
@MetRonnie gotta rerender :)

@conda-forge-admin, please rerender

**Comment by ngam** [comment]
There is no reason this pkg isn't noarch. Please fix.

see my previous PR: https://github.com/conda-forge/cylc-flow-feedstock/pull/31

**Comment by oliver-sanders** [comment]
> @conda-forge/cylc-flow are you still unsure about noarch?

After a chat with the conda-forge folks we think the `_unix` psudo dependency will allow us to make Cylc a noarch package whilst disabling non-unix builds.

Addresses #30

**Comment by MetRonnie** [comment]
Thanks @ngam for the tips, I am still new to conda-forge.

Note to self: https://conda.io/projects/conda-build/en/latest/resources/define-metadata.html#subpackage-requirements

@conda-forge-admin, please rerender

**Comment by MetRonnie** [comment]
@conda-forge-admin, please rerender

**Review by ngam — APPROVED** [review]

**Comment by ngam** [comment]
Looks so much better now. Thanks @MetRonnie @oliver-sanders.

Watch out for some testing --- ideally, you should take test this pkg locally to ensure everything is fine (I am 99.9% sure it will be okay). You can safely ignore if people install it on Windows; they may get some errors about deps, etc., but that should be fine, nothing really to do with your package.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -56,12 +61,15 @@ outputs:
   - name: {{ name }}
     build:
       script: python -m pip install .[empy,report-timings] --no-deps --ignore-installed -vv
```
Comment (ngam) [inline recipe/meta.yaml:63]:
```suggestion
      script: {{ PYTHON }} -m pip install .[empy,report-timings] --no-deps --ignore-installed -vv
```

I think they prefer this now...

File: recipe/meta.yaml [hunk 2]
```diff
@@ -33,12 +35,15 @@ outputs:
   - name: {{ name }}-base
     build:
       script: python -m pip install . --no-deps --ignore-installed -vv
```
Comment (ngam) [inline recipe/meta.yaml:37]:
```suggestion
      script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv
```

**Comment by ngam** [comment]
Also, you could fix https://github.com/conda-forge/cylc-flow-feedstock/issues/28 by removing the username and rerendering again. Maybe @MetRonnie should be added as well?

File: recipe/meta.yaml (hunk 1 above)
Comment (oliver-sanders) [reply recipe/meta.yaml:63]: For some reason that doesn't work in the `outputs` section.

Comment (oliver-sanders) [reply recipe/meta.yaml:63]: (changed here - https://github.com/conda-forge/cylc-flow-feedstock/pull/29/files)

File: recipe/meta.yaml (hunk 2 above)
Comment (ngam) [reply recipe/meta.yaml:37]: pls ignore this suggestion

File: recipe/meta.yaml (hunk 1 above)
Comment (ngam) [reply recipe/meta.yaml:63]: Okay, pls ignore :)

**Comment by oliver-sanders** [comment]
I've asked about removing maintainers on the conda-forge Gitter, I didn't get any definitive answer so in the absence of a documented route I guess we would just remove the entry from maintainers, re-render and hope for the best.

**Comment by oliver-sanders** [comment]
> You can safely ignore if people install it on Windows

The whole `__unix` thing should cause `conda` to refuse to install on Windows due to unsatisfiable dependencies which saves the issue of broken builds 🤞.

**Comment by ngam** [comment]
> I didn't get any definitive answer so in the absence of a documented route I guess we would just remove the entry from maintainers, re-render and hope for the best

Yes, this is the way to do it, imo. Also, you should fee

**Review by hjoliver — APPROVED** [review]

**Comment by hjoliver** [comment]
(Not merging in case you want to address the maintainer change here @MetRonnie, as suggested above).

**Comment by MetRonnie** [comment]
@conda-forge-admin, please rerender

**Review by oliver-sanders — APPROVED** [review]
Looks good

**Comment by oliver-sanders** [comment]
Going in...

**Pull request merged**